website/src/home/content/contributing.md

```diff
@@ -0,0 +1,33 @@
+# Contributing To Compa
+
+Hey! Thanks for considering contributing to Compa. As you already know, Compa is an open-source project and relies on contributions from people like _you_ to function.
+
+Whether code or lesson notes or a helpful comment in a discussion thread, **all** contributions are welcome.
+
+## Code of Conduct
+
+Similar to other open-source communities, we have a [Code of Conduct](https://compa.so/code-of-conduct) which we expect all contributors to follow. In simple terms, don't be an asshole, it will not be tolerated.
+
+## Crowd Sourcing Ethics
+
+We want to ensure that all the content on Compa is of the highest quality. We expect all contributors to follow these guidelines:
+
+- **Be respectful**: We're all here to learn and help each other. Please be respectful to everyone.
+- **Be accurate**: Ensure that the information you're providing is accurate and up-to-date.
+- **Be helpful**: If you're contributing to a discussion, ensure that your comments are helpful and add value to the conversation.
+- **Be original**: If you're contributing code, ensure that it's your own work and not copied from somewhere else. In the event that you must use someone else's work, ensure that you have the necessary permissions to do so and give credit.
+
+Note that repeated violations of these guidelines will result in a ban from the platform.
+
+## Types of Contributions
+
+There are many ways to contribute to Compa. Here are a few examples:
+
+- **Lesson Notes**: If you're a student and have notes from your classes, you can contribute them to Compa. This will help other students who are studying the same subject.
+- **Code**: If you're a developer, you can contribute code to Compa. This could be a new feature, a bug fix, or even a typo correction.
+- **Documentation**: If you're good at writing, you can contribute to the documentation. This could be a new guide, a tutorial, or even a correction to an existing document.
+- **Discussion**: If you're not a developer or a writer, you can still contribute by participating in discussions. You can help answer questions, provide feedback, or even just say hi.
```

Comment:
+ Design/art suggestions

Reply:
added design/art to types of contributions